### PR TITLE
Add deviation threshold to feeds

### PR DESCRIPTION
**Pull request by rupurt**
https://feeds-ui-ak.herokuapp.com/

Screen Shot 2020-02-20 at 8 21 47 AM

### CODE REVIEW

File: feeds/package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   },
   "dependencies": {
     "antd": "^3.23.3",
+    "bignumber.js": "^9.0.0",
```
Comment (hellobart) [inline feeds/package.json]: What do you think about using ethersjs bignum instead?

Comment (rupurt) [reply feeds/package.json]: Does it support arbitrary precision decimals? I was under the impression that it's only for integer arithmetic.

Comment (hellobart) [reply feeds/package.json]: Unfortunately no, Ethers uses `bn.js`. 
How about this: `ethers.utils.formatUnits(ethers.utils.parseUnits("0.01").mul(100), 18)`

Comment (rupurt) [reply feeds/package.json]: I like that's more confusing as percentages are naturally arbitrary precision. I had also considered just using the float in percentage base. i.e.

`0.005` -> `0.5`

**Pull request merged**